easyInterface/Diffraction/DataClasses/DataObj/Experiment.py

```diff
@@ -48,6 +54,20 @@
     }
 }
 
+POLAR_DETAILS = {
```

Comment:
I would propose to use a full name here, POLARIZATION_DETAILS, because POLAR sounds more like polar coordinates.

Reply:
Modified